### ISSUE TITLE
Followed channels autoexpand reversed on left side chat when logged in

### ISSUE THREAD

**Issue by YalithKBK** [night/betterttv#4142]
BetterTTV version 7.2.86 (Aug 31, 2020)
Chrome version 85.0.4183.102 (Official Build) (64-bit)

I do NOT have "Auto Expand Followed Channels List" enabled, although problem persists whether it is enabled or not. I have "Left Side Chat" enabled.

When NOT moused over, the follower pane background expands, covering part of the right side of the screen (see below). If I mouse over the follower pane, it returns to the regular strip. It does NOT expand on mouse-over whether or not I have that setting enabled.

This is not an issue if I am not logged in to twitch.

I have cleared my cache and cookies before opening this issue.

![image](https://user-images.githubusercontent.com/8658787/92995939-c6476300-f4d5-11ea-89fd-c282d4d5649e.png)
![image](https://user-images.githubusercontent.com/8658787/92995972-f55dd480-f4d5-11ea-8e04-bb8c55325a46.png)

**Comment by night** [comment]
I am not able to reproduce this behavior with Left Side Chat enabled locally. If you use additional browser extensions, try disabling them.

**Comment by Maddosaurus** [comment]
I can confirm this issue with BTTV 7.2.87 running on Win 10 x64.  
Same problem: If I am logged in to Twitch, the left side chat has spacing issues - if I re-open the tab in Incognito Mode, everything works as intended (see screenshot 1 vs 2).  
I have deactivated all browser addons except BTTV and cleared the cache as well as cookies - I also tried to clear all addon settings.

![grafik](https://user-images.githubusercontent.com/8026915/94993134-20948a80-058f-11eb-9951-5b69b87639c9.png)
![grafik](https://user-images.githubusercontent.com/8026915/94993154-44f06700-058f-11eb-94e0-020941a50875.png)
